tests/ui/test_ui_tools.py

```diff
@@ -2274,9 +2274,8 @@ class TestStreamButton:
     def test_text_content(self, mocker,
                           is_private, expected_prefix,
                           width, count, short_text, caption='caption'):
-        mocker.patch(STREAMBUTTON + ".mark_muted")
         controller = mocker.Mock()
-        controller.model.muted_streams = {}
+        controller.model.is_muted_stream.return_value = False
```

Comment:
I'm not sure why this works, but `mocker.patch('controller', 'model.is_muted', return_value=False) doesn't.